.github/workflows/angular.yml

```
@@ -32,9 +32,37 @@ on:
     branches:
       - '*'
 jobs:
+  build-matrix:
+    runs-on: ubuntu-20.04
+    outputs:
+      matrix: ${{ steps.build.outputs.matrix }}
+      client: ${{ steps.build.outputs.client }}
+      server: ${{ steps.build.outputs.server }}
+      any: ${{ steps.build.outputs.any }}
+      workflow-angular: ${{ steps.build.outputs.workflow-angular }}
+      workflow-react: ${{ steps.build.outputs.workflow-react }}
+      workflow-vue: ${{ steps.build.outputs.workflow-vue }}
+    steps:
+      - name: 'SETUP: Checkout jhipster-bom'
+        uses: actions/checkout@v3
+        with:
+          path: jhipster-bom
+      - name: 'SETUP: Checkout generator-jhipster'
+        uses: actions/checkout@v3
+        with:
+          repository: 'jhipster/generator-jhipster'
+          path: generator-jhipster
+          fetch-depth: 2
+          ref: main
+      - name: 'Build matrix'
+        id: build
+        uses: ./generator-jhipster/.github/actions/build-matrix
+        with:
+          workflow-samples-file: angular
```

Changes detection won't work here.
```suggestion
          generator-jhipster-directory: ${{ github.workspace }}/generator-jhipster
          skip-changes-detection: true
          workflow-samples-file: angular
```